src/SideMenu/SideMenu.tsx

```diff
@@ -88,7 +92,7 @@ class SideMenu extends Component {
 										primaryTypographyProps={
 											this.state.location === item.link
 												? { color: 'primary', style: { fontWeight: 500 } }
-												: null
+												: undefined
```

Comment:
Again: I'm interested in the preference of `undefined` over `null`, since `null` is more explicit.

Reply:
This has also something to do with the types. It should be a `<value>` OR `undefined` because the prop is optional. If we pass in `null` it is an actual 'value' which is not a valid value in this case.